app/DynamicsAdapter/DynamicsAdapter.Web.Test/Mapping/MappingProfileTest.cs

```diff
@@ -84,6 +85,107 @@ public void SSG_SearchApiRequest_should_map_to_PersonSearchRequest_correctly()
             Assert.AreEqual(2, personSearchRequest.Identifiers.Count);
         }
 
+        [Test]
+        public void Employment_should_map_to_SSG_Employment_correctly()
+        {
+            var employment = new Employment()
+            {
+                Employer = new Employer()
+                {
+                    ContactPerson = "Person",
+                    Name = "Name",
+                    OwnerName = "OwnerName",
+                    
```

Comment:
Shall we add some Phone data here?

Reply:
The employment map will not handle phone, we will have to map phone directly .. it's going to a different entity

Reply:
But we have to test if Employer has an array of Phones, what's the output?